src/components/main/Main.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import Filter from '../../containers/filter/Filter';
+import {Movie} from '../movie/Movie';
+import {Pagination} from '../pagination/Pagination';
+import {MovieI} from '../../core/interface';
+import './Main.scss';
+
+
+interface MainPropsI {
+    movies: Array<MovieI>;
+    total: number;
+    offset: number;
+    limit: number;
+    setSize: (val: number) => void;
+    setPage: (val: number) => void;
+}
+
+export class Main extends React.Component<MainPropsI, {}> {
+    render() {
+        if (this.props.movies.length > 0) {
+            return (
+                <div className="main">
+                    <div className="total"><b>{this.props.total}</b> {this.props.total === 1 ? 'movie' : 'movies'} found</div>
```

Comment:
Better use scss for styles instead of a tag.
`'movie' + this.props.total === 1 ? '' 's'}`